actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -292,7 +292,7 @@ def _enforce_open_redirect_protection(location, allow_other_host:)
           ActiveSupport::Notifications.instrument("open_redirect.action_controller",
             location: location,
             request: request,
-            stack_trace: caller,
+            stacktrace: caller,
```

Comment:
This was released in 8.1, so I'm not sure its worth breaking backwards compatibility to remove the underscore

Reply:
We can remove the underscore in the new structured event subscriber method, but not here. The notification must remain the same.